arviz/stats/stats.py

```diff
@@ -986,8 +986,8 @@ def summary(
     extend: boolean
         If True, use the statistics returned by ``stat_funcs`` in addition to, rather than in place
         of, the default statistics. This is only meaningful when ``stat_funcs`` is not None.
-    credible_interval: float, optional
-        Credible interval to plot. Defaults to 0.94. This is only meaningful when ``stat_funcs`` is
+    hpd_interval: float, optional
+        hpd interval to plot. Defaults to 0.94. This is only meaningful when ``stat_funcs`` is
```

Comment:
Maybe you can fix here the issue you opened for `summary` docs by replacing "hpd interval to plot" by something like "hpd interval to compute and report"?

Reply:
Done Thanks!